src/main/java/eu/dissco/backend/domain/annotation/batch/AnnotationEvent.java

```diff
@@ -1,8 +1,9 @@
 package eu.dissco.backend.domain.annotation.batch;
```

Comment:
I think you can remove the /batch subdirectory and move everything into the domain/annotation package